### PR TITLE
fix: compliance docs update

### PR DESCRIPTION
**Pull request by vinckr**
- fix: threat models doc update
- fix: security section sidebar
- chore: fix typos
- chore: missing feature

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/kratos/concepts/security.mdx [hunk 1]
```diff
@@ -3,210 +3,40 @@ id: security
...
+Ory follows [Digital Identity Guidelines](https://pages.nist.gov/800-63-3/) established by the
+[National Institute of Standards and Technology (NIST)](https://www.nist.gov/).  
+These guidelines are accompanied by a [FAQ](https://pages.nist.gov/800-63-3/) that provides additional information and answers to
+common questions.
 
-## Account enumeration
+## Brute-force defenses
 
-:::warning
+Ory Network provides Ory Identities with protection against brute force attacks by rate-limiting requests to API public endpoints,
```
Comment (tomekpapiernik) [inline docs/kratos/concepts/security.mdx:27]:
```suggestion
Ory Network provides Ory Identities with protection against brute-force attacks by rate limiting requests to API public endpoints,
```
Adjective b4 noun = hyphen
rate limiting = no hyphen (I know Grammarly tells you to use a hyphen, but "rate limiting" here is not describing a quality of "requests")

File: docs/kratos/concepts/security.mdx [hunk 2]
```diff
@@ -3,210 +3,40 @@ id: security
...
+for example login and registration endpoints.
 
-This feature is a work in progress and is tracked as [kratos#133](https://github.com/ory/kratos/issues/133).
+When self-hosting the Ory Kratos Identity Server, it is the responsibility of the administrator to implement and manage
+rate-limiting or other measures to ensure the security of the network.
```
Comment (tomekpapiernik) [inline docs/kratos/concepts/security.mdx:31]: Mixing Ory Network and self-hosted instances

File: docs/kratos/concepts/security.mdx [hunk 3]
```diff
@@ -3,210 +3,40 @@ id: security
...
-It doesn't yet work as documented!
+## Account takeover defenses
 
-:::
+The [settings flow](../self-service/flows/user-settings) implements account takeover defenses by requiring the current password
+before allowing the user to change it. This makes it impossible for an attacker to change the password without knowing the
+existing password.
 
-## Digital identity guidelines
+## Password Policy
```
Comment (tomekpapiernik) [inline docs/kratos/concepts/security.mdx:39]:
```suggestion
## Password policy
```

File: docs/kratos/concepts/security.mdx [hunk 4]
```diff
@@ -3,210 +3,40 @@ id: security
...
 
-There is no one standard to digital identity. Ory Identities follows emerging frameworks and guidelines such as:
-[Digital Identity Guidelines established by the National Institute of Standards and Technology (NIST)](https://pages.nist.gov/800-63-3/)
-(and a follow-up [FAQ](https://pages.nist.gov/800-63-3/)) .
+To learn more about setting up a secure password policy, please refer to the
```
Comment (tomekpapiernik) [inline docs/kratos/concepts/security.mdx:41]:
```suggestion
To learn more about setting up a secure password policy, refer to the
```

No need to be overly polite

File: docs/kratos/concepts/security.mdx [hunk 5]
```diff
@@ -3,210 +3,40 @@ id: security
...
+When self-hosting the Ory Kratos Identity Server, it is the responsibility of the administrator to implement and manage
+rate-limiting or other measures to ensure the security of the network.
 
-It doesn't yet work as documented!
+## Account takeover defenses
 
-:::
+The [settings flow](../self-service/flows/user-settings) implements account takeover defenses by requiring the current password
+before allowing the user to change it. This makes it impossible for an attacker to change the password without knowing the
+existing password.
```
Comment (tomekpapiernik) [inline docs/kratos/concepts/security.mdx:37]: Technically changing passwords is possible only when users have a privileged session. To get a privileged session, users must authenticate. Do you think that it's a good idea to describe this?

**Review by tomekpapiernik — CHANGES_REQUESTED** [review]

File: docs/kratos/concepts/security.mdx [hunk 6]
```diff
@@ -3,210 +3,40 @@ id: security
...
 
-Will be addressed in a future release.
+There is no universally accepted standard for digital identity.  
+Ory follows [Digital Identity Guidelines](https://pages.nist.gov/800-63-3/) established by the
+[National Institute of Standards and Technology (NIST)](https://www.nist.gov/).  
+These guidelines are accompanied by a [FAQ](https://pages.nist.gov/800-63-3/) that provides additional information and answers to
+common questions.
 
-## Account enumeration
+## Brute-force defenses
```
[the remark this answers is not in the document]
Comment (vinckr) [reply docs/kratos/concepts/security.mdx:25]: It is defense **against** brute-force. 

If I write **anti-brute force defense**, I am defending against anti-brute force attacks (?)

File: docs/kratos/concepts/security.mdx [hunk 7]
```diff
@@ -3,210 +3,40 @@ id: security
...
 ---
 
-:::warning
+Threat modeling is essential for securing a system like Ory Identities. This document provides guidelines for identifying and
+analyzing threats and implementing security measures for Ory Identities and self-hosted Ory Kratos Identity Server.
 
-Please be aware that this chapter is still work in progress. Not all mitigation strategies have been implemented yet in Ory
-Identities!
+Running software that stores personal information carries risk. It is important to identify threats and understand the system's
+attack surface, the likelihood, and the impact of an attack.
```
[the remark this answers is not in the document]
Comment (vinckr) [reply docs/kratos/concepts/security.mdx:10]:
```suggestion
Working with and administering software that stores personal information carries risk. It is important to identify threats and understand the system's
attack surface, the likelihood, and the impact of an attack.
```

?

File: docs/kratos/concepts/security.mdx [hunk 8]
```diff
@@ -3,210 +3,40 @@ id: security
 title: Threat models and security profiles
 ---
 
-:::warning
+Threat modeling is essential for securing a system like Ory Identities. This document provides guidelines for identifying and
+analyzing threats and implementing security measures for Ory Identities and self-hosted Ory Kratos Identity Server.
```
[the remark this answers is not in the document]
Comment (vinckr) [reply docs/kratos/concepts/security.mdx]: I think in this instance it is OK 😅

File: docs/kratos/concepts/security.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (tomekpapiernik) [reply docs/kratos/concepts/security.mdx:25]: Yes, that's what it does. But looking at this after a few days it doesn't really sound right. The best solution would be to use a heading that is constructed a bit differently than the rest: 

`Defenses against brute-force attacks`

[the remark this answers is not in the document]
Comment (vinckr) [reply docs/kratos/concepts/security.mdx]: done!

**Review by tomekpapiernik — APPROVED** [review]
Looks good to me now. Go ahead and merge at your discretion. :)

**Pull request merged**